.github/workflows/node.yml

```diff
@@ -40,5 +41,97 @@ jobs:
       - name: Build
         run: yarn build
 
-      - name: Test
-        run: yarn test
\ No newline at end of file
+      - name: Lint and test
+        run: yarn test
+
+  validate-prod-dependencies:
+    name: Validate production dependencies
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    timeout-minutes: 15
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install Omniorb 4
+        run: sudo apt-get install libomniorb4-dev
+      - name: Prepare Environment
+        run: yarn
+        env:
+          CI: true
+      - name: Validate production dependencies
+        run: |
+          if ! git log --format=oneline -n 1 | grep -q "\[ignore-audit\]"; then
+            yarn validate:dependencies
+          else
+            echo "Skipping audit"
+          fi
+        env:
+          CI: true
+
+  dockerhub-release:
+    name: Make dockerhub release
+    runs-on: ubuntu-latest
+    needs: [ build-and-test, validate-prod-dependencies ]
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get the Docker tag
+        id: docker-tag
+        uses: yuya-takeyama/docker-tag-from-github-ref-action@2b0614b1338c8f19dd9d3ea433ca9bc0cc7057ba
+        with:
+          remove-version-tag-prefix: false
+      - name: Determine images to publish
+        id: image-tags
+        # TODO - image needs changing...
```

Comment:
Was this my TODO that got copied from somewhere or something you added?

Reply:
Comes from you. Not sure its relevant here ... or there!